CMakeLists.txt

```diff
@@ -81,7 +81,7 @@ if (WIN32)
 endif (WIN32)
 
 # Define the main library.
-add_library(snappy SHARED
+add_library(Snappy SHARED
```

Comment:
Why? This is something other packages may have already started relying on, and I really see no advantage of making it uppercase (and therefore inconsistent with `-lsnappy`).

Even worse, if you change this, then packages that already `target_link_libraries(snappy)` will silently switch from linking the target to implicit `-lsnappy`.